index.html

```diff
@@ -26,6 +26,224 @@ <h1>Hey There. I’m Alex</h1>
+                   </div>
+                </div>
+
+                <div class="grid-item languages">
+                <h2><img class="imageTitle1" alt="languages" src="imgs/languages.svg"></h2>
+                  <ul class="languages_list1">
+                         <li>Javascript</li>
+                         <li>Ruby</li>
+                         <li>Html</li>
+                         <li>Css</li>
+                     </ul>
+  
+                     <h2><img class="imageTitle2" alt="languages" src="imgs/Frameworks.svg"></h2>
+                     <ul class="languages_list1">
+                         
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                     
+                    <h2><img class="imageTitle3" alt="languages" src="imgs/skills.svg"></h2>
+                     <ul class="languages_list1">
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                  
+              </div>
+
+              <div class="grid-item">
+                  <div class="description">
+                      <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+                  </div>
+              </div>  
+          
+              <div class="grid-item">
+                <div class="forms">
+                    <form action="#" method="GET">
+                        <input type="text" placeholder="Full name">
+                        <input type="email" placeholder="Email adress">
+                        <textarea placeholder="Enter text here"></textarea>
+                        <div class="submit_button">
+                        <input type="submit" value="Get in touch">
+                        </div>
+
+                    </form>
+                </div>
+            </div>
+
+
+            
+            <div class="My_footer">
+                   
+                <ul class="social_icon2">
+                    <li><a href="#"><img class="icon" alt="social_icon" src="imgs/github2.svg"></a></li>
+                    <li><a href="#"><img class="icon" alt="social_icon" src="imgs/linkedin2.svg"></a> </li>
+                    <li><a href="#"><img class="icon" alt="social_icon" src="imgs/thund2.svg"></a></li>
+                    <li><a href="#"><img class="icon" alt="social_icon" src="imgs/twitter2.svg"></a></li>
+                    <li><a href="#"><img class="icon" alt="social_icon" src="imgs/m2.svg"></a></li>
+              </ul>
+              <div class="line2">
+                </div>
+            </div>
+              </div>
+        </div>
```

Comment:
- [OPTIONAL] Kindly this is another section insted of using a div for the section you can use `<footer>... </footer> `. 
- The design of the footer is not match to the figma template because of the `margin: 26%; height: 5px; ` you have giving to the `line2 class` make the padding around , Kindly fix that and let's see your project get approve 
- Kindly the icons must be under the line2.

Your design
![Screenshot from 2022-07-02 22-50-57](https://user-images.githubusercontent.com/77842237/177014519-6987f79d-935a-4211-9518-c664b3ed3ab5.png)
 Figma design
![Screenshot from 2022-07-02 22-51-21](https://user-images.githubusercontent.com/77842237/177014531-18aeb2c2-3a0c-4cf0-8cb9-2c66e242f361.png)